examples/nlp/language_modeling/conf/megatron_gpt_config.yaml

```diff
@@ -114,3 +117,9 @@ model:
       warmup_steps: 500
       constant_steps: 50000
       min_lr: 2e-5
+
+    # Megatron O2-style half-precision
+    megatron_amp_o2: False # Enable O2-level automatic mixed precision using master parameters
```

Comment:
A small nitpick but should this be named `megatron_amp_O2` ?

Reply:
Technically O2 should be right. But, I see that only lower case is used in the config? @ericharper  Do you suggest using O2 instead of o2?